script.js

```diff
@@ -0,0 +1,55 @@
+const slides = document.querySelectorAll(".img-container");
```

Comment:
can you create slider using JS object, not directly via HTML
so like
const images = ['image1.png', 'image2.png'......]
and then render these image
Else everything should be working same @Rishab-innovative